### PR TITLE
Fix typo in configure examples

### PR DESCRIPTION
**Pull request by pallxk**

### CODE REVIEW

File: awscli/examples/configure/get/_examples.rst [hunk 1]
```diff
@@ -24,12 +24,12 @@ The following commands would have the corresponding output::
...
 
     $ aws configure get profile.testing.aws_access_key_id
-    default_access_key
+    testing_access_key
 
     $ aws configure get preview.cloudsearch
     true
 
     $ aws configure get preview.does-not-exist
-    $
```
Comment (JordonPhillips) [inline awscli/examples/configure/get/_examples.rst]: This is showing that there is no output, resulting in the prompt immediately appearing. I would prefer to keep this.

**Comment by JordonPhillips** [comment]
Thanks for the PR! Taking a look.

File: awscli/examples/configure/get/_examples.rst (hunk 1 above)
Comment (pallxk) [inline awscli/examples/configure/get/_examples.rst]: I made the change as I saw there is always an empty line between prompts. I ran my test with PS1 set to `"\n$ "`. I'm also okay to keep this and amend my commit.

**Comment by pallxk** [comment]
Rebased and amended to include most relevant changes to this PR.

**Comment by joguSD** [comment]
Cool. Thanks for incorporating that feedback. Merging.

**Pull request merged**